conda_smithy/templates/azure-pipelines-osx.yml.tmpl

```diff
@@ -6,10 +6,11 @@ jobs:
   strategy:
     maxParallel: 8
     matrix:
-    {%- for config, platform in configs | sort %}
+    {%- for config, platform, upload in configs | sort %}
     {%- if platform.startswith('osx') %}
       {{ config }}:
-        CONFIG: {{ config}}
+        CONFIG: {{ config }}
+        UPLOAD_PACKAGES: {{ upload }}
```

Comment:
@mariusvniekerk, is there a way to make this a global env variable?

Reply:
that looks pretty global to me?

Reply:
so we could use a `variables` block